ocaml/typing/typemod.ml

```diff
@@ -3885,7 +3889,9 @@ let package_units initial_env objfiles target_cmi modulename =
       raise(Error(Location.in_file mli, Env.empty,
                   Interface_not_compiled mli))
     end;
-    let name = Compilation_unit.name modulename in
+    let name =
+      Compilation_unit.name modulename |> Compilation_unit.Name.to_global_name
```

Comment:
Turns out it's not actually wrong: this is in `package_units`, so there cannot be arguments (since we're building a pack). But it's clearer with `Compilation_unit.to_global_name_without_prefix` anyway so I'm changing it.